ssl/ssl_conf.c

```diff
@@ -520,6 +520,22 @@ static int cmd_DHParameters(SSL_CONF_CTX *cctx, const char *value)
     return rv > 0;
 }
 #endif
+
+static int cmd_RecordPadding(SSL_CONF_CTX *cctx, const char *value)
+{
+    size_t block_size = (size_t)atoi(value);
+
+    if (block_size > 1 &&
```

Comment:
The documentation above says that 0 or 1 turns off padding (which seems correct behaviour to me). However this implementation fails on padding of 0.